packages/language-kit/CHANGELOG.md

```diff
@@ -0,0 +1,65 @@
+# Change Log
```

Comment:
We don't need to modify this file; it is auto-added by the CI.

Reply:
Removed.